components/Nav.js

```diff
@@ -15,16 +15,18 @@ export default ({ file }) => {
   const npmUrl = 'https://npmjs.com/' + name;
 
   const File = ({ meta, parent }) => html`
-    <li key=${meta.path} style=${{ order: 1 }}>
+    <li key=${meta.path}>
       ${FileIcon}
-      <a onClick=${() => pushState(`?${name}@${version}${meta.path}`)}
+      <a
+        href=${`/?${name}@${version}${meta.path}`}
+        onClick=${() => pushState(`?${name}@${version}${meta.path}`)}
```

Comment:
Same here. No need for onclick

Reply:
How do you prevent the page from loading without `onClick`? We added `href` to make it tab-able but `onClick` for client side routing.

Reply:
Addressed in commit `d1b37e5`